docs/requirements.md

```diff
@@ -107,6 +107,7 @@ You must manually create the databases required by Codacy on the PostgreSQL serv
     CREATE DATABASE activities WITH OWNER=codacy;
     CREATE DATABASE hotspots WITH OWNER=codacy;
     CREATE DATABASE listener WITH OWNER=codacy;
+    CREATE DATABASE crow WITH OWNER=codacy;
```

Comment:
Good catch! I had forgotten about the necessity of having this db now that crow is part of self-hosted too!
👍